formbricks/formbricks
# feat: resolvable notes

## What does this PR do?

Add the ability to resolve individual notes in the response summary section.

Also updated the UI/UX when users interact with the note. Users are now able to hide the textarea to see more notes! And updated the text color of the note header to match the background and icon.

Switched to use the minimize icon.

Fixes #390 

https://www.loom.com/share/be6316aaae9b43d49504eb4f882968ec

## Type of change




## How should this be tested?



Go to any survey response summary page with a response. Open the note for a response. Create a new note. Hover over a note to see the edit and check icon. Click the edit icon to edit the note. Click the check icon to resolve the note. The resolved note should disappear.

## Checklist
